docs/ui/components/Home/sections/QuickStart.tsx

```diff
@@ -52,6 +53,50 @@ export function QuickStart() {
             Start Tutorial
           </HomeButton>
         </GridCell>
+        <GridCell
+          className={mergeClasses(
+            'bg-palette-pink3 border-palette-pink7 relative z-0 min-h-[164px] dark:bg-palette-pink3',
+            'max-md-gutters:min-h-[200px]'
+          )}>
+          <RouterLogo
+            className={mergeClasses(
+              'size-20 absolute right-7 bottom-7 border-[5px] rounded-xl p-3',
+              'stroke-[0.01rem] stroke-palette-pink8 text-palette-pink8 bg-palette-pink4 border-palette-pink5'
+            )}
+          />
+          <RawH2 className="!text-palette-pink11 relative z-10 !text-lg">
+            Get what's the best in file-based routing with Expo Router
```

Comment:
```suggestion
            Discover the benefits of file-based routing with Expo Router
```

Let's update this to the above. I think using the word "what" here makes it sound weird.